### DISCUSSION TITLE
System Fonts

### DISCUSSION THREAD

**Discussion by chrissm79** [tamagui/tamagui#589] [Q&A]
I've created a theme with the font property set to `font: {}`. However, this causes an error when I attempt to use the `Button` component `Cannot read property 'size' of undefined` which points [here](https://github.com/tamagui/tamagui/blob/b77e4dbf63e913bc2bac8e4292d1462eb40a3468/packages/font-size/src/getFontSize.ts#L47). It looks like the function is trying to obtain the default size of a defined font, and since I don't have one it's throwing. 

I prefer to use the default system fonts on both iOS/Android but the examples provided showcase using the theme with the inter font. How can I provide the default fonts to the configuration?

**Comment by sandulat** [comment]
It would also be great if custom fonts were not a requirement, but rather an option. It makes sense for native apps trying to achieve the truly "native" feeling.

**Reply by salvinoto** [reply]
Wait so I'm having this error, I don't mind using custom fonts but can't figure out how to even use a custom one. I loaded Inter using react native, but that's about it. And the error still occurs

**Comment by dacapoo** [comment]
have you found a solution for this problem ? it's really annoying that you can not define native fonts for a native app.
